gateway/src/apicast/policy/cors/apicast-policy.json

```diff
@@ -46,7 +46,7 @@
         }
       },
       "allow_origin": {
-        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. https://example.com) or '*'. If left blank, the value of the 'Origin' request header will be used.",
+        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. https://example.com) or '*'. If left blank, the value of the 'Origin' request header will be used. In case of valid regexp, and if it matches with Origin header value, the value will be set to the Origin Value. In case of does not match, the header will not set at all.",
```

Comment:
I think this messages could be improved if we add some examples and a bit of markup (I don't know if html is being escaped):

```suggestion
        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. <code>https://example.com</code>) or '*'. If left blank, the value of the 'Origin' request header will be used. <br> In order to allow more than one origin it is possible to use a regular expression, if it matches with Origin header value, the value will be set to the Origin Value. In case it does not match, the header will not set at all. Example: <code>(api|web).test.com</code> wil match both <code>api.test.com</code> and <code>web.test.com</code>",
```
It will look like:
<img width="522" alt="Screenshot 2021-01-14 at 11 08 41" src="https://user-images.githubusercontent.com/13486237/104576830-fdcb4500-5658-11eb-83f2-c2fbde4a8243.png">

Reply:
WDYT @thomasmaas ?

Reply:
Yes but I think it has to be plain text, not HTML.

Reply:
In that case, we can simply use quotes instead of <code> tags:
```suggestion
        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. "https://example.com") or '*'. If left blank, the value of the 'Origin' request header will be used. In order to allow more than one origin it is possible to use a regular expression, if it matches with Origin header value, the value will be set to the Origin Value. In case it does not match, the header will not set at all. Example: "(api|web).test.com" wil match both "api.test.com" and "web.test.com".",
```
It will look like this:

> <img width="533" alt="Screenshot 2021-01-14 at 11 48 00" src="https://user-images.githubusercontent.com/13486237/104581180-83052880-565e-11eb-92a1-14633481a453.png">